.gitignore

```diff
@@ -65,3 +65,6 @@ dist
 /driver/*.mod
 /driver/*.mod.c
 /driver/.*.cmd
+
+# Visual Studio Code files
+.vscode
```

Comment:
not a project member but this should usually be split out into a separate commit

Reply:
I lost count of projects and the companies that I worked for using git and I have never in my life have seen a project that asked me to deliver the .gitignore file in a separate commit.

Reply:
Agreed with @whot , that should be a separate commit since it has nothing to do with the device support. I'd even say since you are adding two separate devices those should be separate commits, but no worries about that.